.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -101,37 +101,56 @@ if [ "$STATUS" == "SUCCESS" ]; then
 	exit 0
 fi
 
-curl --header "Accept: application/json" --header "Authorization: Bearer $TEAMCITY_TOKEN" http://ci-oss.hashicorp.engineering/app/rest/testOccurrences?locator=build:$ID --output tests.json -L
-set +e
-
 # This is an intentionally dumb list; if something is removed and re-added with
 # the same name, we'll still catch it. If that ends up causing noise, we can do
 # something more clever.
-NEW_TESTS=$(git diff --unified=0 | { grep -oP '(?<=^\+func )Test\w+(?=\(t \*testing.T\) {)' || test $? = 1; } | tr '\n' ' ')
-ALL_TESTS=$(cat tests.json | jq -r '.testOccurrence | map(.name) | .[]')
+NEW_TESTS=$(git diff --unified=0 HEAD~1 | { grep -oP '(?<=^\+func )TestAcc\w+(?=\(t \*testing.T\) {)' || test $? = 1; } | tr '\n' ' ')
```

Comment:
Why do we need to track new tests? From what I've seen they show up as failed tests as they don't have cassettes recorded

Reply:
Skipped tests or GA-only tests do not show up as failed tests

Reply:
I had forgotten to link the issue - more details there: https://github.com/hashicorp/terraform-provider-google/issues/9586